app/serializers/v1/customer_serializer.rb

```diff
@@ -75,6 +76,17 @@ def billing_configuration
       configuration
     end
 
+    def shipping_address
```

Comment:
Could you add a test  for this new field :pray: :wink:

Reply:
done